docs/source/gbd2017_models/risks/correlated_risks/index.rst

```diff
@@ -0,0 +1,1199 @@
+.. _2017_risk_models:
+
+====================
+Risk Correlation Proposal
+====================
+
+Background and motivation
+----------------------
+
+At the individual level, exposure to risk factors are likely to be correlated. Several examples include high body mass index and high fasting plasma glucose, tobacco smoking and alcohol use, and childhood height and weight. Vivarium takes population exposure prevalence estimates by age/sex/year/location and determines the if a simulant is exposed to the risk factor such that the prevalence within the simulation matches the GBD estimate. The exposure status for multiple risk factors of the individuals within Vivarium should be adjusted so that they are correlated across those risks.
+
+
+Although correlated risk factor exposure may be important in a variety of contexts, we are proposing using childhood height and weight as a testing and motivating example. Childhood growth is an important metric for how children are developing. Children with low height and weight for their age are at higher risk of getting and dying from infectious diseases like diarrhea, lower respiratory infections, and measles. Height and weight should be correlated between children and each child should have dynamic but correlated height and weight over time as they age.  
+
+Risk exposure correlation
+------------------------------
+
+The proposed method to introduce correlation in risk factor exposure is by finding the Spearman’s correlation coefficient between two or more risks and sampling from a multivariate normal distribution when assigning a position on an exposure distribution. This plot shows data from a joint distribution reflecting the variance and covariance between height and weight at one month using data from a cohort study. The top panel shows the values for the height and weight and the bottom panel shows the percentiles within the distributions (propensity values).
```

Comment:
I'd like some explanation about why Spearman's correlation coefficient is the right thing to look at, and how you use it to create a multivariate normal distribution.  It would be good to add a link explaining Spearman's correlation (e.g. https://en.wikipedia.org/wiki/Spearman%27s_rank_correlation_coefficient) and any other important concepts that might be unfamiliar to people.